src/web/basics/button/button.scss

```diff
@@ -15,10 +15,13 @@
     // default size: standard
     padding: 0 16px;
     height: 36px;
-    line-height: 36px;
+    line-height: 34px;
     min-width: 68px;
     font-size: 14px;
 
+    // always have a border to prevent box-sizing vertical spacing issues
+    border: 1px solid color(adsk-blue);
```

Comment:
Nice. This has been bugging me.

Consider setting `border-color: transparent`, then you don't have to override it for hover and focus.